frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTableConfig.tsx

```diff
@@ -99,12 +99,29 @@ const generateTableHeaders = (
     canAddOrDeletePolicy?: boolean;
     tableType?: string;
   },
-
+  policiesList: IPolicyStats[] = [],
   isPremiumTier?: boolean,
   isSandboxMode?: boolean
 ): IDataColumn[] => {
   const { selectedTeamId, tableType, canAddOrDeletePolicy } = options;
-
+  // Figure the time since the host counts were updated.
+  let timeSinceHostCountUpdate: string;
+  policiesList.some(function (policyItem) {
```

Comment:
My instinct is to write this as a standalone function, then call it to set the value, but not critical.
```
const getTimeSinceHostCountUpdate = (policies: IPolicyStats[]): string => {...};
...
const timeSinceHostCountUpdate = getTimeSinceHostCountUpdate(policiesList);
```